util/regexp-assemble/data/932240.data

```diff
@@ -0,0 +1,27 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regexp_assemble/.
+
+##! Rule 932240 (Generic RCE Unix command evasion detection)
+##!
+##! Conceptually, we want to match:
+##! - at least one char
+##! - at least one character from an evasion technique
+##! -
+##!+ i
+
+##!$ \b
```

Comment:
I don't think we need this.

```suggestion
##!$ \b
```

Reply:
I've removed this, I wonder why is not showing.